lib/widgets/color_container.dart

```diff
@@ -29,14 +29,13 @@ class ColorContainer extends StatelessWidget {
                 c.CupertinoColors.label,
                 context,
               ),
-              fontSize: 20,
             ),
           ),
           Container(
-            padding: EdgeInsets.symmetric(vertical: 12, horizontal: 12),
+            padding: EdgeInsets.symmetric(vertical: 8, horizontal: 8),
```

Comment:
8 seems a bit small. what do you say?

Reply:
8 Looks better but Let me change it to 10